eucaconsole.spec

```diff
@@ -43,6 +43,8 @@ Source2:        %{name}
 Source3:        %{name}.sysconfig
 
 Patch0:         console.default.ini.patch
+Patch1:         gevent.patch
+Patch2:         eucaconsole.requires.patch
```

Comment:
Patches should begin with the project name, e.g. `eucaconsole-4.3.0-gevent.patch`